data/test_environment.py

```diff
@@ -0,0 +1,25 @@
+import sys
+
+REQUIRED_PYTHON = "python3"
+
+
+def main():
+    system_major = sys.version_info.major
+    if REQUIRED_PYTHON == "python":
+        required_major = 2
```

Comment:
python2 is dead, please put it out of its misery

Reply:
removed!